lib/ansible/cli/vault.py

```diff
@@ -76,6 +76,8 @@ def init_parser(self):
         create_parser = subparsers.add_parser('create', help='Create new vault encrypted file', parents=[vault_id, common])
         create_parser.set_defaults(func=self.execute_create)
         create_parser.add_argument('args', help='Filename', metavar='file_name', nargs='*')
+        create_parser.add_argument('--skip-tty-check', default=False, help='allows editor to be opened when no tty attached',
```

Comment:
dont make this a new switch, but a new configuration entry would work (ansible.cfg and env var to set)